doc(release): release notes for collect 24.04 (#3450)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs/version-24.04/releases/centreon-os.mdx

## Centreon Collect

### 24.04.4

Release date: `July 29, 2024`

<details>
  <summary>Bug fixes</summary>

- [Broker] If a service status was not 0, 1 or 2, it was written in RRD as an empty value. This raised an error in librrd. Now we send 'U' to RRD, which is a value that librrd accepts.
- [Broker] The default log level for Broker is now error instead of info (except for the core and config logs, which are info).
- [Engine] The default log level for Engine is now error instead of info (except for the core and config logs, which are info).

> - [Engine] The output of external commands is no longer truncated.

Reviewer: ```suggestion
- [Engine] The output of external commands is no longer truncated after a backslash.
```

## Centreon Collect

### 24.04.4

> Release date: `July 27, 2024`

Author: ```suggestion
Release date: `July 29, 2024`
```

<details>
  <summary>Bug fixes</summary>

- [Broker] If a service status was not 0, 1 or 2, it was written in RRD as an empty value. This raised an error in librrd. Now we send 'U' to RRD, which is a value that librrd accepts.
- [Broker] The default log level for Broker is now error instead of info (except for the core and config logs, which are info).
- [Engine] The default log level for Engine is now error instead of info (except for the core and config logs, which are info).

> - [Engine] fix(engine): output of external commands are sometimes truncated

Reviewer: ```suggestion
- [Engine] The output of external commands is no longer truncated.
```

> - [Engine] fix (common): anomaly config files are open in readonly mode

Reviewer: ```suggestion
- [Engine] Configuration files for Anomaly Detection are now opened in read-only mode.
```

> - [Engine] fix (common): all logger configs are taken into account (runtime and otl)

Reviewer: ```suggestion
- [Engine] All log file configurations are now taken into account (including runtime and otl).
```

- [Engine] If a service contained several metrics with the same name, some of them were overwritten. Now during the parsing, a warning log explains the issue and only the first occurrence of the metric is kept.

> - [Engine] If a service contains several times the same metric name, some of them were overwritten. Now during the parsing, a warning log explains the issue and only the first occurence of the metric is kept.

Reviewer: ```suggestion
```
